sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -342,7 +342,8 @@ abstract class OffsetWindowFunction
   extends Expression with WindowFunction with Unevaluable with ImplicitCastInputTypes {
   /**
    * Input expression to evaluate against a row which a number of rows below or above (depending on
-   * the value and sign of the offset) the current row.
+   * the value and sign of the offset) the current row or the first row of the entire window
```

Comment:
`...  the starting row. (current row if isRelative=true, or the first row of the window frame otherwise)`

Reply:
OK